apps/transformer/lib/transform/add_timestamp.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Transform.AddTimestampField do
+  @moduledoc """
+  `Transform.Step` impl to add a timestamp field to a dataset. Timestamps will be stored in ISO-8601 format.
+
+  ## Configuration
+  * `name` - Required. Field name or path to store the timestamp under. Ex: `"timestamp"` or `["timestamp"]`
```

Comment:
what is the difference between naming it "timestamp" vs ["timestamp"]

Reply:
As I understand it, Transforms can accept either a string for a field name or a list of atoms for an access path. This is covering both possibilities.

Reply:
if it's a list of atoms then it should be `"timestamp"` or `[:timestamp]`

Reply:
Well, I copied the documentation from the delete and move field transformations, where the arguments are either strings or lists of strings. Should I change the @moduledoc in all three?

Reply:
The list allows you to nest. We should probably differentiate the example to make the point.

`"timestamp"` and `["key1", "key2", "key3"]`

Reply:
I changed the documentation as Johnson suggested.